src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -45,28 +46,6 @@ static VisualStudioInstanceFactory()
         public VisualStudioInstanceFactory()
         {
             AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolveHandler;
-            AppDomain.CurrentDomain.FirstChanceException += FirstChanceExceptionHandler;
-        }
-
-        private static void FirstChanceExceptionHandler(object sender, FirstChanceExceptionEventArgs eventArgs)
```

Comment:
I want to write to the test framework's test output, which requires threading through the xunit interface. I don't want to create "yet another log file" for kicks.